src/Tizen.NUI.Components/Controls/Picker.cs

```diff
@@ -0,0 +1,570 @@
+                {
+                    foreach (TextLabel textLabel in itemList)
+                    {
+                        if (pickerScroller) pickerScroller.Remove(textLabel);
+                        Utility.Dispose(textLabel);
+                    }
+
+                    itemList = null;
+                }
+
+                if (pickerScroller != null)
+                {
+                    Remove(pickerScroller);
+                    Utility.Dispose(pickerScroller);
+                    pickerScroller = null;
+                }
+
+                Remove(upLine);
+                Utility.Dispose(upLine);
+                Remove(downLine);
+                Utility.Dispose(downLine);
+            }
+
+            base.Dispose(type);
+        }
+
+        /// <summary>
+        /// An event emitted when Picker value changed, user can subscribe or unsubscribe to this event handler.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler<ValueChangedEventArgs> ValueChanged;
+
+        //TODO Fomatter here
+
+        /// <summary>
+        /// The values to be displayed instead of numbers.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ReadOnlyCollection<String> DisplayedValues
+        {
+            get
+            {
+                return displayedValues;
+            }
+            set
+            {
+                displayedValues = value;
+
+                needItemUpdate = true;
+                displayedValuesUpdate = true;
+
+                UpdateValueList();
+            }
+        }
+        
+        /// <summary>
+        /// The Current value of Picker.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int CurrentValue
+        {
+            get
+            {
+                return currentValue;
+            }
+            set
+            {
+                if (currentValue == value) return;
+
+                if (currentValue < minValue) currentValue = minValue;
+                else if (currentValue > maxValue) currentValue = maxValue;
+
+                currentValue = value;
+
+                UpdateCurrentValue();
+            }
+        }
+
+        /// <summary>
+        /// The max value of Picker.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int MaxValue
+        {
+            get
+            {
+                return maxValue;
+            }
+            set
+            {
+                if (maxValue == value) return;
+                if (currentValue > value) currentValue = value;
+                
+                maxValue = value;
+                needItemUpdate = true;
+
+                UpdateValueList();
+            }
+        }
+
+        /// <summary>
+        /// The min value of Picker.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int MinValue
```

Comment:
It seems that crash happens if MinValue <= 0

Reply:
I missed this corner case thank you.